docs/devel/cherry-picks.md

```diff
@@ -64,6 +64,13 @@ hack/cherry_pick_pull.sh upstream/release-3.14 98765
 This will walk you through the steps to propose an automated cherry pick of pull
  #98765 for remote branch `upstream/release-3.14`.
 
+#### Labeling
+
+Ensure your PR contains the label `cherrypick-candidate`.
```

Comment:
Hmm, I'm not the only one confused about these labels, re: https://github.com/kubernetes/kubernetes/pull/23543#issuecomment-202511995, so I think the process is (correct me of I'm wrong):

During code slush times:
* On master, PRs labeled as `cherrypick-candidate` are triaged and batched 
* On master, PRs accepted for batching are labeled `cherrypick-approved`

During other times:
* Anyone can propose a cherrypick to a target branch using the script
* On the target branch, PRs land with no labels (though [maybe they should labeled](https://github.com/kubernetes/kubernetes/pull/23543#issuecomment-202623507) with `cherrypick-candidate` for tracking purposes? Or maybe that's just redundant)
* If the PR doesn't immediately have a `cherrypick-approved` label, it is labeled `do-not-merge`

cc @eparis